autodeploy/routers/security.py

```diff
@@ -0,0 +1,28 @@
+
+from pydantic import BaseModel
+from typing import Optional
+
+from fastapi import APIRouter
+from fastapi import Depends, FastAPI, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+
+from security.scheme import oauth2_scheme, fake_users_db, get_user, fake_decode_token, fake_hash_password
+from security.model import User, UserInDB
+
+router = APIRouter(
+    prefix="/token",
+    tags=["security"],
+    responses={}
+)
+
+@router.post("/")
+async def login(form_data: OAuth2PasswordRequestForm = Depends()):
```

Comment:
it depends on nothing?

Reply:
Yeah! it doesnt. more info here : https://fastapi.tiangolo.com/tutorial/security/simple-oauth2/

Reply:
cool :)